### ISSUE TITLE
reflect.StructOf is missing

### ISSUE THREAD

**Issue by adranwit** [tinygo-org/tinygo#3599]
When trying compile code with reflect.StructOf I see
: undefined: reflect.StructOf

- creating go struct on the fly can not be achieved without  reflect.StructOf call

**Comment by dgryski** [comment]
Correct.  TinyGo does not have the ability to create types at runtime at the moment.  The best we can do is stub the call to panic if called, and hope nobody calls it..

**Comment by adranwit** [comment]
Would it be possible to add it, I am happy to contribute (any direction to kick start it), I see that currently for struct type definition the following is used structNamesSidetable, structTypesSidetable, I would guess that the latter are used for unnamed inline types  it is possible at runtime to operated on structTypesSidetable, (reallocate and update varint size of these ) ?

**Comment by dgryski** [comment]
This is actually a large chunk of work.
